uobtheatre/bookings/schema.py

```diff
@@ -275,6 +275,7 @@ def get_queryset(cls, queryset, info):
     class Meta:
         model = Booking
         filterset_class = BookingFilter
+        exclude = ("_transfered_to",)
```

Comment:
We don't seem to expose transferred_to on booking nodes, only from - we should expose the transferred_to property